analyzers/tests/ITs.JsonParser.Test/JsonParsingTest.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace SonarAnalyzer.Net8.Test;
+
+[TestClass]
+public class JsonParsingTest
+{
+    //TODO: Add tests during implementation
+    [TestMethod]
+    public void NewIssueReports_Test()
```

Comment:
Right now I cannot cover anything, since it's just two dummy methods in a top-level main (which is not accessible).

I think the PR is already quite big and it sets up the skeleton of the two new projects, I wouldn't go further than this.
Next PR will add 50% of the functionality needed+some tests.